config/vendor-api-v1.yml

```diff
@@ -471,6 +473,20 @@ components:
           description: Other personal or professional issues relevant to the application
             which are not covered in the form
           example: I am expecting to be called for jury service in the next 12 months
+        safeguarding_issues_status:
+          type: string
+          description: Status of candidate's response to the safeguarding issues declaration
+          enum:
+          - not_answered_yet
```

Comment:
I don't think they'll ever get this?

Reply:
You are right, on providers side it can't be not_answered_yet.